app/api/helpers/tasks.py

```diff
@@ -452,3 +452,13 @@ def export_speakers_pdf_task(self, event_id):
         result = {'__error': True, 'result': str(e)}
 
     return result
+
+
+@celery.task(base=RequestContextTask, name='delete.translations', bind=True)
+def delete_translations(self, zip_filename, uuid_literal, temp_dir_path):
+    try:
+        os.remove(os.path.join(temp_dir_path, zip_filename))
+        temp_dir_path.close()
+    except Exception as e:
+        result = {'__error': True, 'result': str(e)}
+        logging.error(result['result'])
```

Comment:
This should be a human readable string, like 'Error while parsing the json response'

Reply:
How can we obtain this?

Reply:
Should I be logging the traceback `traceback.format_exc()`?